sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -185,14 +148,24 @@ export interface QueueOptions {
 }
 
 // @public
-export interface ReceiveBatchOptions extends OperationOptions {
+export interface ReceiveBatchOptions extends OperationOptions, WaitTimeOptions {
 }
 
 // @public
-export type ReceivedMessage = Omit<ServiceBusMessage, "complete" | "abandon" | "defer" | "deadletter" | "prototype">;
+export interface ReceivedLockedMessage extends ReceivedMessage {
```

Comment:
Just putting them out.. 
Three types
- ServiceBusMessage
- ReceivedLockedMessage   ->  ReceivedMessageWithLock (How does this sound???)
- ReceivedMessage

Reply:
ReceivedMessageWithLock -> gold!